src/controllers/__tests__/messagesController.test.ts

```diff
@@ -59,6 +59,12 @@ const mockedUser: User = {
   wallet_token: "123hexToken" as WalletToken
 };
 
+const mockedDefaultParameters = {
+  continuationToken: undefined,
```

Comment:
why `continuationToken`?

Reply:
Fixed in https://github.com/pagopa/io-backend/pull/830/commits/15434c00d547ab9ae09e3b48719e133e4cfbcb22